src/runtime/runtime.cs

```diff
@@ -328,6 +320,29 @@ internal static void Initialize(bool initSigs = false)
             AssemblyManager.UpdatePath();
         }
 
+        private static IntPtr Get_PyObject_NextNotImplemented()
+        {
+            IntPtr globals = PyDict_New();
+            if (PyDict_SetItemString(globals, "__builtins__", PyEval_GetBuiltins()) != 0)
```

Comment:
Can `PyEval_GetBuiltins()` return an error?

Reply:
From the docs it doesn't look like it can (https://docs.python.org/3/c-api/reflection.html#c.PyEval_GetBuiltins).

Reply:
The globals dict is needed, but we don't seem to be using the `__builtins__` in the string. Do we really need this line?

Reply:
If you don't setup a `__builtins__ ` or set `object` explicit, `object` may not be found.